examples/kitchen-sink/src/pages/static-mdx-query.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { graphql, StaticQuery } from "gatsby";
+import MDXRenderer from "gatsby-mdx/mdx-renderer";
+
+export default function anotherPage() {
+  return (
+    <StaticQuery
+      query={graphql`
+        query {
+          mdx(id: { eq: "c9610a63-e336-5fdc-b2ca-abe0e4143e22" }) {
+            code
+          }
+        }
+      `}
+      render={data => {
+        return (
+          <div>
+            <MDXRenderer>{data.mdx.code}</MDXRenderer>
```

Comment:
what happens if MDXRenderer is in an imported component like:

```js
<div><MyOtherThing>{data.mdx.code}</MyOtherThing></div>
```

```js
export const MyOtherThing ({children}) => <MDXRenderer>{children}</MDXRenderer>
```

Reply:
A very good question.

Reply:
Changed this to use Context to pass down the scope.